coalib/output/ConsoleInteraction.py

```diff
@@ -312,17 +313,18 @@ def print_results_formatted(log_printer,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
-                                        **result.__dict__))
+                                        **format_args))
                 continue
 
             for range in result.affected_code:
+                format_args['affected_code'] = range
```

Comment:
ahhh omg sorry... that's the problem when built-in names are reused :3
Yes this makes totally sense :+1: :+1:
sorry again :D